5.2/5.2.2/debian/buildout.cfg

```diff
@@ -12,6 +12,10 @@ parts +=
   mrbob
   plonesite
 
+eggs +=
+  psycopg2
+  RelStorage
```

Comment:
```suggestion
  RelStorage[mysql,postgresql,oracle]
```

Reply:
Oh, I didn't know I could add more than one extra this way, thanks :)